module.go

```diff
@@ -136,6 +154,11 @@ graphql: {
 	multipartForm: {
 		uploadMaxSize: (int | *1.5M) & > 0
 	}
+	limitQueryAmountMiddleware: {
+		enable: bool | *false
```

Comment:
Please add a config section to the readme file explaining the middleware and its two options.